docs/modules/development/pages/rest/resources.adoc

```diff
@@ -12,8 +12,14 @@ This service is especially useful in conjunction with the Measurements API.
 | `/resources`                            | Retrieve the full tree of resources in the system (expensive, use with care).
 | `/resources/\{resourceid}`              | Retrieve the tree of resources starting with the named resource ID.
 | `/resources/fornode/\{nodecriteria}`    | Retrieve the tree of resources for a node, given its database ID or `foreign-source:foreign-ID` tuple.
+| `/resources/select`                     | Select partial node resources by specifying node ids (database ids or external ids), node filter rules, subresource names, and string property names using the attributes `nodes`, `filterRules`, `nodeSubresources`, and `stringProperties`, respectively. All attributes can have comma separated lists of values.
 |===
 
+The `/resources/select` endpoint returns only selected parts of node resources:
+
+* If the `nodeSubresources` attribute is not set then a shallow copy of all node subresources (i.e. without nested content) is included. Otherwise the selected node subresources are included and some or all of their string properties.
```

Comment:
```suggestion
* If the `nodeSubresources` attribute is not set, then a shallow copy of all node subresources (i.e., without nested content) is included. 
Otherwise, the selected node subresources are included and some or all of their string properties.
```